liberty-managed/src/main/java/io/openliberty/arquillian/managed/WLPManagedContainer.java

```diff
@@ -1437,7 +1445,7 @@ private String getServerOutputDir() throws IOException {
     * @throws IOException
     */
    private String getServerConfigDir() throws IOException {
-	   String serverConfigDir = getWlpUsrDir() + "servers/" + containerConfiguration.getServerName();
+	   String serverConfigDir = getWlpUsrDir() + "/servers/" + containerConfiguration.getServerName();
```

Comment:
Your PR seems to superseed mine?

https://github.com/OpenLiberty/liberty-arquillian/pull/68

Reply:
Yes, it seems so. Sorry for the inconvenience.

Reply:
No worries, I'll just close mine.